docs/notebooks/annoytutorial.ipynb

```diff
@@ -269,16 +277,16 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "('orbits', 0.6323515474796295)\n",
-      "('changes,', 0.6309329569339752)\n",
-      "('dr.', 0.6305853128433228)\n",
-      "('terrorism,', 0.6300898194313049)\n",
-      "('creditors', 0.6264415979385376)\n"
+      "('signature', 0.5921074748039246)\n",
+      "('\"dangerously', 0.5920691192150116)\n",
```

Comment:
This looks like bad preprocessing. Any reason not to simply use `simple_preprocess`?